.gitignore

```diff
@@ -48,6 +48,10 @@ tmp
 
 # Distribution folders
 dist/
+dist-node/
+dist-browser/
+*/**/browsertest/browsertest.js
+*/**/browsertest/browsertest.min.js
 
```

Comment:
We also need to add some cypress auto-generated files here. See https://github.com/LiskHQ/lisk-elements/blob/1.0.0/.gitignore#L26-L29